charts/linkerd-control-plane/templates/destination.yaml

```
@@ -264,6 +264,7 @@ spec:
         - --default-policy={{.Values.policyController.defaultAllowPolicy}}
         - --log-level={{.Values.policyController.logLevel | default "linkerd=info,warn"}}
         - --log-format={{.Values.controllerLogFormat}}
+        - --probe-networks={{.Values.policyController.probeNetworks | join ","}}
```

Note that I haven't actually tested it but I agree, per @mateiidavid's testing, that we should omit the flag when it's empty.

```suggestion
{{- if not empty .Values.policyController.probeNetworks }}
        - --probe-networks={{.Values.policyController.probeNetworks | join ","}}
{{- end }}
```